products/arangodb

```diff
@@ -0,0 +1,33 @@
+---
+title: ArangoDB
+category: db
+permalink: /arangodb
+releasePolicyLink: https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy
+changelogTemplate: "https://docs.arangodb.com/stable/release-notes/version-__RELEASE_CYCLE__/"
+releaseLabel: "ArangoDB __RELEASE_CYCLE__"
+eolWarnThreshold: 121
+releases:
+-   releaseCycle: "3.10"
+    latest: "3.10.11"
```

Comment:
Could you also document `latestReleaseDate` for all releases ?

Reply:
I don't know where to get that information unfortunately

Reply:
On https://github.com/arangodb/arangodb/tags (using tag dates). Tag dates may not be the exact release date, but they are usually pretty close.